src/page-components/Home/index.tsx

```diff
@@ -7,24 +8,28 @@ import { PageMain } from "@components/Page-Main";
 import { PageSection } from "@components/Page-Section";
 import { Button } from "@components/Button";
 import { SmIcons } from "@components/SM-Icons";
+import { FlipCard } from "@components/Flip-card";
+
+import { HorizontalScroller } from "@components/Horizontal-scroller";
 import {
   ButtonColor,
   ButtonVariant,
 } from "@components/Button/constants";
 
-
-
 import styles from "./styles.module.css";
+import { Skills } from "types";
 
 type Props = {
   summary?: string;
   start?: string;
   name?: string;
   occupation?: string;
   section_two_title?: string;
```

Comment:
use camel case for naming `secondSectionTitle`. Also name should be properly descriptive